### DISCUSSION TITLE
Cant get CardDav to activate in Accounts

### DISCUSSION THREAD

**Discussion by plp05045** [blind-coder/rcmcarddav#457] [Q&A] [answered]
I have filled out the fields to have cardav auto propagate for email users but no matter what I do it does not show up on the accounts. I have attached my config file.

**Comment by mstilkerich** [comment, marked as answer]
Mind the `/*` and `*/` which denote start and end of a multiline comment. All your config is part of such a comment. Furthermore your config is in the middle of the example of the sample file so it is syntactically broken too.

**Reply by plp05045** [reply]
That will do it! Thanks so much!
